api/errors/errors.go

```diff
@@ -28,9 +28,6 @@ var ErrGetValueForKey = errors.New("get value for key error")
 // ErrGetKeyValuePairs signals an error in getting the key-value pairs of a key for an account
```

Comment:
We also need to export account nonce

Reply:
exported since core's AlteredAccount is used

Reply:
:pray: